src/org/rascalmpl/library/lang/c90/syntax/C.rsc

```diff
@@ -30,11 +30,11 @@ syntax Statement
 
 syntax Expression 
 	= variable: Identifier 
-	| @category="Constant" HexadecimalConstant 
-	| @category="Constant" IntegerConstant 
-	| @category="Constant" CharacterConstant 
-	| @category="Constant" FloatingPointConstant 
-	| @category="Constant" StringConstant 
+	| @category="constant" HexadecimalConstant 
```

Comment:
Constant doesn't exist in semantic tokens.

They should be `"number"` and `"string"` for example.